routers/customer_manager/workday.py

```diff
@@ -8,7 +8,10 @@
 MICROSERVICE_URL = "http://localhost:8001"
 
 @router.post("/workdays/workday/")
-async def create_workday(workday_request: CreateWorkdayRequest):
+async def create_workday(workday_request: CreateWorkdayRequest, request: Request):
+    from auth.auth0_client import get_current_user as check_if_logged
```

Comment:
this import shouldn't be here; if you're getting a circular import error, then move the invalid function somewhere else (like to other file)